presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -2,6 +2,32 @@
 Hive Security Configuration
 ===========================
 
+Authorization
+=============
+You can enable authorization checks for queries that are running in Presto and
+using the Hive connector, by setting the ``hive.security`` property in
+``hive.properties`` file. This property takes three values:
+
+================================================== ============================================================
+Property Value                                     Description
+================================================== ============================================================
+``none`` (default value)                           No authorization checks are enforced, thus allowing any
+                                                   operation.
+
+``read-only``                                      Operations that read data or metadata, such as ``SELECT``,
+                                                   are permitted, but none of the operations that write data or
+                                                   metadata, such as ``CREATE``, ``INSERT`` or ``DELETE``, are
+                                                   allowed.
+
+``sql-standard``                                   Users are permitted to perform the operations as long as
+                                                   they have the required privileges as per the SQL standard.
+                                                   In this mode, Presto enforces the authorization checks for
+                                                   queries based on the privileges defined in Hive metastore.
```

Comment:
I added the following line:
"Hive connector now supports ``GRANT`` and ``REVOKE`` statements using which you
can grant and revoke privileges to users or roles defined in Hive metastore.
See :doc:`/sql/grant` and :doc:`/sql/revoke` for their usages."